jobs/openvpn-client/templates/etc/user.erb

```diff
@@ -0,0 +1,2 @@
+<% if_p('username') do | v |  %><%= p('username') %><% end %>
+<% if_p('password') do | v | %><%= p('password') %><% end %>
```

Comment:
Conceptually, if username is specified, should password be required, too? Just wondering if we should do `if_p('username')` to wrap both `p(...)` statements to try and have it error if `password` is not provided.

Reply:
Technically openvpn doesn't require the password to be supplied as it will prompt for it on startup. In a bosh environment without an operator to prompt it would make sense to have it error if username is supplied but password isn't.